tests/letstest/scripts/test_leauto_upgrades.sh

```diff
@@ -26,6 +26,17 @@ else
     # 0.33.x is the oldest version of letsencrypt-auto that works on Fedora 29+.
     INITIAL_VERSION="0.33.1"
 fi
+
+# If we're on RHEL 8, the initial version of certbot-auto will fail until we do
```

Comment:
We should create issue to mark the fact we need to remove this code once the next version of Certbot is released.

Reply:
Created https://github.com/certbot/certbot/issues/7301.